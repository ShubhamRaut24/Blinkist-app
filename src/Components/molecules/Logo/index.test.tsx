import React from 'react';
import LogoComponent from ".";
import renderer from 'react-test-renderer';


test('Logo Component', () => {
    const component = renderer.create(
            <LogoComponent height= '30' url= '/assets/logo.png' name= 'Blinkist'></LogoComponent>
    );
    expect(component).toMatchSnapshot();
});