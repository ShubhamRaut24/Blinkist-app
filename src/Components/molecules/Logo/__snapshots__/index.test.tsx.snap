// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component 1`] = `
<div
  className="MuiBox-root css-70qvj9"
>
  <img
    alt="not found"
    className="MuiBox-root css-qc5z28"
    src="/assets/logo.png"
  />
  <p
    className="MuiTypography-root MuiTypography-body1 css-1vrfq90-MuiTypography-root"
  >
    Blinkist
  </p>
</div>
`;
